### PR TITLE
Add deselected text in audio track when deselecting a candidate

### PR DESCRIPTION
**Pull request by carolinemodic**
Adds "Deselected" to the beginning of the a choice's audio track right after it is unselected. After 1 second this is removed. I tried removing it right away and it weirdly seemed to work reliably with the space bar but not with the enter key which was strange, and it was impossible to test so I think having the timeout makes the most sense.

Feel free to let me know if this isn't a good spot to add the tests for this. I tried to do it in the component unit test but since I can't control the updateVote function there it wasn't possible. 

https://zube.io/votingworks/vxsuite/c/1783

### CODE REVIEW

**Review by eventualbuddha — APPROVED** [review]
LGTM with one small change.

File: apps/bmd/src/components/YesNoContest.tsx [hunk 1]
```diff
@@ -57,6 +57,13 @@ const YesNoContest = ({ contest, vote, updateVote }: Props) => {
   const [overvoteSelection, setOvervoteSelection] = useState<
     Optional<YesOrNo>
   >()
+  const [deselectedVote, setDeselectedVote] = useState('')
+
+  useEffect(() => {
+    if (deselectedVote !== '') {
+      setTimeout(() => setDeselectedVote(''), 1000)
+    }
```
[the remark this answers is not in the document]
Comment (carolinemodic) [reply apps/bmd/src/components/YesNoContest.tsx:65]: Oh yes, added that at the end and totally forgot about the other place haha good catch

File: apps/bmd/src/components/CandidateContest.tsx [hunk 2]
```diff
@@ -313,6 +324,12 @@ const CandidateContest = ({ contest, parties, vote, updateVote }: Props) => {
                   const party =
                     candidate.partyId &&
                     findPartyById(parties, candidate.partyId)
+                  let prefixAudioText = ''
+                  if (isChecked) {
+                    prefixAudioText = 'Selected, '
```
Comment (beausmith) [inline apps/bmd/src/components/CandidateContest.tsx:329]: Space after comma is unnecessary as it is in the template. Remove one of the spaces in both templates.

File: apps/bmd/src/AppContestSingleSeat.test.tsx [hunk 3]
```diff
@@ -81,4 +81,24 @@ it('Single Seat Contest', async () => {
...
     'false'
   )
+
+  // Deselect the first candidate
+  fireEvent.click(getByText(candidate0))
+
+  // Check that the aria label has deselected added
+  expect(
+    getByText(candidate0).closest('button')?.getAttribute('aria-label')
+  ).toContain('Deselected, ')
```
Comment (beausmith) [inline apps/bmd/src/AppContestSingleSeat.test.tsx:91]: Is there a "startsWith"?

File: apps/bmd/src/AppContestSingleSeat.test.tsx [hunk 4]
```diff
@@ -81,4 +81,24 @@ it('Single Seat Contest', async () => {
...
+  // Check that other candidates do not have deselected added
+  expect(
+    getByText(candidate1).closest('button')?.getAttribute('aria-label')
+  ).not.toContain('Deselected, ')
```
Comment (beausmith) [inline apps/bmd/src/AppContestSingleSeat.test.tsx:95]: Trailing spaces aren't really necessary.

File: apps/bmd/src/AppContestSingleSeat.test.tsx [hunk 5]
```diff
@@ -81,4 +81,24 @@ it('Single Seat Contest', async () => {
...
+
+  // After 1 second the deselected label is removed
```
Comment (beausmith) [inline apps/bmd/src/AppContestSingleSeat.test.tsx:97]: I lean towards not using comments that are verbatim what the code is doing. I fear they will become out of date with the code; say if a search and replace is done on the code.

File: apps/bmd/src/components/CandidateContest.tsx [hunk 6]
```diff
@@ -142,6 +143,15 @@ const CandidateContest = ({ contest, parties, vote, updateVote }: Props) => {
     }
   }, [vote.length, updateContestChoicesScrollStates])
 
+  useEffect(() => {
+    if (deselectedCandidate !== '') {
+      const timer = setTimeout(() => {
+        setDeselectedCandidate('')
+      }, 1000)
```
Comment (beausmith) [inline apps/bmd/src/components/CandidateContest.tsx:150]: Can we use a smaller value and have it still work?

**Review by beausmith — COMMENTED** [review]
Looks great! Just a couple of nits and questions.

File: apps/bmd/src/AppContestSingleSeat.test.tsx (hunk 3 above)
Comment (carolinemodic) [reply apps/bmd/src/AppContestSingleSeat.test.tsx:91]: No :(

**Pull request merged**